sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -869,8 +874,12 @@ case class ShowTablesCommand(
       val database = tableIdent.database.getOrElse("")
       val tableName = tableIdent.table
       val isTemp = catalog.isTempView(tableIdent)
-      val information = partition.simpleString
-      Seq(Row(database, tableName, isTemp, s"$information\n"))
+      val infoValue = if (conf.getConf(SQLConf.LEGACY_KEEP_COMMAND_OUTPUT_SCHEMA)) {
```

Comment:
seems worthy of having `def legacyKeepCommandOutputSchema: Boolean = getConf(LEGACY_KEEP_COMMAND_OUTPUT_SCHEMA)`